python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -127,13 +128,19 @@ def __init__(self, provider: NodeProvider):
         self.worker_ids: List[NodeID] = []
         # The head node (node kind "head")
         self.head_id: Optional[NodeID] = None
+        # Map of replica IDs to worker nodes in each replica
+        self.replicas_to_nodes = defaultdict(list)
```

Comment:
Can we add some comments for (1) the definition of `replica` and (2) `replica` is currently for KubeRay only?

Reply:
Done.